queries/models/planejamento/staging/aux_voronoi_stops.py

```
@@ -0,0 +1,143 @@
+# -*- coding: utf-8 -*-
+
+# ----------------------------------------------------------------
+# IMPORTAÇÃO DE BIBLIOTECAS
+# ----------------------------------------------------------------
+# Bibliotecas padrão e geoespaciais
+import geopandas as gpd
```

Ruff (0.12.2)

`geopandas` imported but unused

Remove unused import: `geopandas`

(F401)